examples/cpp_api/nullable_attribute.cc

```diff
@@ -178,11 +178,23 @@ void read_array() {
   std::cout << "a2: " << std::endl;
   for (i = 0; i < 4; ++i) {
     if (a2_validity_buf[i] > 0) {
-      std::cout << "{ ";
-      std::cout << std::to_string(a2_data[i * 2]);
-      std::cout << ", ";
-      std::cout << std::to_string(a2_data[i * 2 + 1]);
-      std::cout << " }";
+      std::cout << "{";
+      int start_idx = a2_off[i] / (sizeof(int));
```

Comment:
Don't we need to divide by `sizeof(uint64_t)` here?

Also below in `end_idx = a2_off[i + 1] / (sizeof(int));`

Reply:
yes, we should divide by sizeof(uint64_t).

Reply:
We reverted it back to the old logic